docker/flags.go

```diff
@@ -32,40 +32,40 @@ var (
 		{"attach", "Attach to a running container"},
 		{"build", "Build an image from a Dockerfile"},
 		{"commit", "Create a new image from a container's changes"},
-		{"cp", "Copy files/folders from a container's filesystem to the host path"},
+		{"cp", "Copy files/folders from a container to a HOSTDIR or to STDOUT"},
 		{"create", "Create a new container"},
 		{"diff", "Inspect changes on a container's filesystem"},
 		{"events", "Get real time events from the server"},
 		{"exec", "Run a command in a running container"},
-		{"export", "Stream the contents of a container as a tar archive"},
+		{"export", "Export a container's filesystem as a tar archive"},
 		{"history", "Show the history of an image"},
 		{"images", "List images"},
-		{"import", "Create a new filesystem image from the contents of a tarball"},
+		{"import", "Import the contents from a tarball to create a filesystem image"},
 		{"info", "Display system-wide information"},
 		{"inspect", "Return low-level information on a container or image"},
 		{"kill", "Kill a running container"},
-		{"load", "Load an image from a tar archive"},
-		{"login", "Register or log in to a Docker registry server"},
-		{"logout", "Log out from a Docker registry server"},
+		{"load", "Load an image from a tar archive on STDIN"},
```

Comment:
Oh, terribly sorry, looks like I missed another one :blush: (actually, this mistake is in the text you copied from, so not your mistake)

`on STDIN` should be `or STDIN` here. And the same typo is in [/docs/reference/commandline/load.md](https://github.com/docker/docker/blob/master/docs/reference/commandline/load.md), and [/man/docker-load.1.md](https://github.com/docker/docker/blob/master/man/docker-load.1.md)

It would be really awesome if you could fix those, but I'm also fine if that would be in a separate PR :heart: